plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -330,8 +330,10 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
 
 				// fundamental resources
-				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes", "pods").RuleOrDie(),
+				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+				rbac.NewRule(ReadWrite...).Groups(legacyGroup).Resources("pods").RuleOrDie(),
```

Comment:
why does the scheduler need create and delete?

Reply:
Doesn't need create, but it needs update for updating pod's annotations and delete for preempting pods. The rest are not needed. I just used "ReadWrite" since it was already defined. I can change it. "Read" + update + delete is enough.

Reply:
also, these are being added unconditionally, even if preemption is not enabled

Reply:
annotations can be updated via pods/status. update pods lets a scheduler change the image the pod is running, which is not something I'd expect a scheduler to be able to do

Reply:
That's great that we can update annotations via pod/status. I will remove the "update" permission. I totally agree with you that updating image is security vulnerability. Actually, @davidopp and I were discussing this yesterday. I am glad that it can be addressed easily.